Doc/reference/compound_stmts.rst

```diff
@@ -157,17 +157,16 @@ The :keyword:`for` statement is used to iterate over the elements of a sequence
    for_stmt: "for" `target_list` "in" `starred_list` ":" `suite`
            : ["else" ":" `suite`]
 
-The expression list is evaluated once; it should yield an iterable object.  An
-iterator is created for the result of the ``starred_list``.  The expression
-list can contain starred elements (``*x, *y``) that will be unpacked in the
-final iterator (as when constructing a ``tuple`` or ``list`` literal). The
-suite is then executed once for each item provided by the iterator, in the
-order returned by the iterator.  Each item in turn is assigned to the target
-list using the standard rules for assignments (see :ref:`assignment`), and then
-the suite is executed.  When the items are exhausted (which is immediately when
-the sequence is empty or an iterator raises a :exc:`StopIteration` exception),
-the suite in the :keyword:`!else` clause, if present, is executed, and the loop
-terminates.
+The starred list expression is evaluated once; it should yield an iterable object.
+If the expression is a sequence, it can contain starred elements (``*x, *y``)
+that will be unpacked, as when constructing a ``tuple`` or ``list`` literal.
```

Comment:
Do we need this sentence? It's not part of the semantics of the for loop itself, but of the `starred_list` expression.

Reply:
gone